pkg/openshift/certgen/templates/master/tmp/ansible/azure-local-master-inventory.yml

```diff
@@ -74,6 +72,10 @@ localmaster:
     template_service_broker_image_name: "IMAGE_TYPE-template-service-broker"
     template_service_broker_version: "vVERSION"
 
+    # NOTE: This is redundant, but required to handle version skew between
+    #       openshift-ansible in Origin and OCP
+    openshift_examples_content_version: "vSHORT_VER"
```

Comment:
is SHORT_VER still defined in this PR?  Looks like it's been dropped?

Reply:
it's not defined explicitly because I can do a bash parameter expansion to obtain the value and the sed statement is still there with `s|SHORT_VER|${VERSION%.*}|g;`

Reply:
thanks, was looking in the wrong place